src/common/uilocale.cpp

```diff
@@ -664,8 +664,21 @@ int wxUILocale::GetSystemLanguage()
         }
     }
 
-    // no info about this language in the database
-    return wxLANGUAGE_UNKNOWN;
+    // no info about the preferred UI language in the database
+    // fall back to default locale
+    return GetSystemLocale();
+}
+
+/*static*/
+int wxUILocale::GetSystemLocale()
+{
+    // Retrieve default wxUILocale via empty wxLocaleIdent
+    wxLocaleIdent emptyLocId;
+    wxUILocale defaultLocale(emptyLocId);
+
```

Comment:
If this is the only reason we need to change `wxUILocale` ctor above, I'd rather find a different solution, i.e. have a function returning default locale ID directly instead.

Reply:
> If this is the only reason we need to change `wxUILocale` ctor above, I'd rather find a different solution, i.e. have a function returning default locale ID directly instead.

Yes, that's the only reason. The default locale is needed here, but we have only method `wxUILocale::UseDefault` to create it - and that's not what we want to do here.